core/src/main/scala/kafka/utils/LogDirUtils.scala

```diff
@@ -18,15 +18,16 @@
 package kafka.utils
 
 import kafka.controller.LogDirEventNotificationHandler
+import kafka.zk.KafkaZkClient
 import scala.collection.Map
 
 object LogDirUtils extends Logging {
 
   private val LogDirEventNotificationPrefix = "log_dir_event_"
   val LogDirFailureEvent = 1
 
-  def propagateLogDirEvent(zkUtils: ZkUtils, brokerId: Int) {
-    val logDirEventNotificationPath: String = zkUtils.createSequentialPersistentPath(
+  def propagateLogDirEvent(zkClient: KafkaZkClient, brokerId: Int) {
```

Comment:
Perhaps we could delete the other two methods in this class since they don't seem to be used.